app/src/main/java/com/celzero/bravedns/adapter/UniversalAppListAdapter.kt

```diff
@@ -86,18 +86,18 @@ class UniversalAppListAdapter(private val context: Context, private val appInfoR
                 GlideApp.with(context).load(context.packageManager.getApplicationIcon(appInfo.packageInfo)).into(b.univWhitelistApkIconIv)
             } catch (e: Exception) {
                 GlideApp.with(context).load(AppCompatResources.getDrawable(context, R.drawable.default_app_icon)).into(b.univWhitelistApkIconIv)
-                Log.e(LOG_TAG, "Application Icon not available for package: ${appInfo.packageInfo}" + e.message, e)
+                Log.e(LOG_TAG_FIREWALL, "Application Icon not available for package: ${appInfo.packageInfo}" + e.message, e)
```

Comment:
As they are part of list adapters its not separated out in a common place as the ImageView are part of viewBinding. Will see if it can be abstracted out.

Reply:
This code was refactored a bunch.